senet/se_googlenet

```diff
@@ -0,0 +1,49 @@
+from torch import nn
+from torchvision.models import googlenet
+from torchsummary import summary
+import warnings
+from collections import namedtuple
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.jit.annotations import Optional, Tuple
+from torch import Tensor
+from senet.se_module import SELayer
+
+class SEGoogLeNet(nn.Module):
+  def __init__(self, num_classes, pretrained, aux_logits=True, transform_input=False):
```

Comment:
I would use 4 spaces for indents as other files.

Reply:
Ok, I will change it.